sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1050,16 +1076,9 @@ private async Task StartPartitionProcessingAsync(string partitionId)
                 }
             }
 
-            // TODO: if reason = Shutdown or OwnershipLost and we got an exception when closing, what should the final reason be?
```

Comment:
This TODO comment was not related with the code you removed.

Reply:
Whoops.   Restored.